tb_plugin/setup.py

```diff
@@ -27,6 +22,11 @@ def get_version():
     "torchvision >= 0.8"
 ]
 
+EXTRAS = {
+    "s3": ["boto3"],
+    "blob": ["azure-storage-blob"]
+}
+
```

Comment:
what about using an existing filesystem manager like fsspec? https://filesystem-spec.readthedocs.io/en/latest/

Reply:
I'd like not introduce additional dependencies for tb_plugin.